capnpc/src/codegen.rs

```diff
@@ -1089,7 +1202,7 @@ fn generate_setter(
                     )));
                     initter_params.push("size: u32");
                     (
-                        Some(fmt!(ctx, "{capnp}::text::Reader<'_>")),
+                        Some("&str".to_string()),
```

Comment:
Hm... it seems a bit surprising to me to have `Text` be the only type where the setter does not accept the corresponding `Reader` type. I understand the reason for it -- otherwise a lot of dependent code would need to be updated with an `.into()` or whatever -- but I don't know if it's the best choice. Perhaps we should get https://github.com/capnproto/capnproto-rust/pull/421 working (see also https://github.com/capnproto/capnproto-rust/issues/419) and allow text-field setters to provide either a `str` or a `text::Reader`.

Reply:
Yes, that was the idea, but it does require some added complexity. Now I'm thinking that it'll be better to avoid that change, because it seems to not actually be required for performance. See https://github.com/capnproto/capnproto-rust/issues/419#issuecomment-1698014044.